clients/osv/src/main/kotlin/OsvApiClient.kt

```diff
@@ -0,0 +1,134 @@
+/*
+ * Copyright (C) 2022 EPAM Systems, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ * License-Filename: LICENSE
+ */
+
+package org.ossreviewtoolkit.clients.osv
+
+import com.jakewharton.retrofit2.converter.kotlinx.serialization.asConverterFactory
+
+import java.util.concurrent.Executors
+
+import kotlinx.serialization.SerialName
+import kotlinx.serialization.Serializable
+import kotlinx.serialization.json.Json
+
+import okhttp3.Dispatcher
+import okhttp3.MediaType.Companion.toMediaType
+import okhttp3.OkHttpClient
+
+import retrofit2.Call
+import retrofit2.Retrofit
+import retrofit2.http.Body
+import retrofit2.http.GET
+import retrofit2.http.POST
+import retrofit2.http.Path
+
+/**
+ * A rest API client for the Google Open Source Vulnerabilities API (OSV.dev), see https://osv.dev/.
+ */
+internal interface OsvApiClient {
+    companion object {
+        const val SERVER_URL_PRODUCTION = "https://api.osv.dev"
+        const val SERVER_URL_STAGING = "https://api-staging.osv.dev"
+        const val BATCH_REQUEST_MAX_SIZE = 1000
+        val JSON = Json { }
```

Comment:
Use `val JSON = Json.Default` instead to resolve and inspection hint. And I'd also add a blank line before this one.